src/main/webapp/mentors.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Get mentors from the mentor servlet.
+ */
+function getMentors() {
+  console.log("entering get mentors function/n");
+  let url = new URL('/mentors', location.protocol + '//' + location.hostname);
+
+  fetch(url).then(response => response.json()).then(mentors => {
+    console.log(mentors);
+    const mentorsContainer = document.getElementById('mentors-container');
+    mentorsContainer.innerHTML = '';
+    for (const mentor of mentors) {
+      mentorsContainer.appendChild(createMentorElement(mentor.name, mentor.description));
+    }
+  });
+}
+
+/**
+ * Creates a mentor container element from input text.
+ * @param {String} name
+ * @param {String} description
+ * @return {<div class="p-1 col-lg-4">} mentorContainer
+ */
+function createMentorElement(name, description) {
+  const mentorContainer = document.createElement('div');
+  mentorContainer.className = 'p-1 col-lg-4';
+
+  const mentorCard = document.createElement('div');
+  mentorCard.className = 'card-holder card h-100 border-primary mb-3';
+
+  const mentorCardBody = document.createElement('div');
+  mentorCardBody.className = 'card-body pb-0';
+
+  const mentorName = document.createElement('h5');
+  mentorName.className = 'card-title text-primary';
+  mentorName.innerHTML = name;
+  mentorCardBody.appendChild(mentorName);
+
+  const mentorDescription = document.createElement('p');
+  mentorDescription.innerHTML = description;
+  mentorCardBody.appendChild(mentorDescription);
+
+  mentorCard.appendChild(mentorCardBody);
+  mentorContainer.appendChild(mentorCard);
```

Comment:
nit: It is usually better to define things when you use them. I think it might be better to create ```mentorCard``` and ```mentorContainer``` at the end, right before they are used or to do all the appending for these at the top before appending the child ```div```s